### PR TITLE
docs: add page on using the command line

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

**Comment by octoper** [comment]
@owenvoke If you want to add this page to the docs navbar you can add it in th `documentation.md` file

**Comment by owenvoke** [comment]
@octoper, any thoughts on where in the sidebar it would be good to go?

**Comment by octoper** [comment]
@owenvoke I think in the Guides section, and change the titel to Command Line Guide

File: command-line.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Command Line Runner
```
Comment (octoper) [inline command-line.md:2]: Change this to: `Command Line Guide`

**Pull request merged**